omod/src/main/java/org/openmrs/module/htmlformentryui/fragment/controller/htmlform/EnterHtmlFormFragmentController.java

```diff
@@ -152,14 +152,14 @@ public void controller(FragmentConfiguration config, UiSessionContext sessionCon
 		//If GP timezone is true, it will convert the visitStartDatetime and visitStopDatetime to UTC and format RFC3339
 		if (visit != null) {
 			if (visit.getStartDatetime() != null) {
-				visitStartDatetime = ui.convertTimezones() ? ui.format(visit.getStartDatetime())
+				visitStartDatetime = ui.convertTimezones() ? ui.dateToISOString(visit.getStartDatetime())
```

Comment:
@icrc-toliveira is the ternary operator still needed?

Reply:
@dkayiwa 
Yup this is indeed correct 
Just pushed a fix